src/qinfer/tests/test_concrete_models.py

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+##
+# test_concrete_models.py: Checks that built-in instances of Model work properly.
+##
+# © 2014 Chris Ferrie (csferrie@gmail.com) and
+#        Christopher E. Granade (cgranade@gmail.com)
+#     
+# This file is a part of the Qinfer project.
+# Licensed under the AGPL version 3.
+##
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+##
+
+## FEATURES ###################################################################
+
+from __future__ import absolute_import
+from __future__ import division # Ensures that a/b is always a float.
+from future.utils import with_metaclass
+
+## IMPORTS ####################################################################
+
+import numpy as np
+from numpy.testing import assert_equal, assert_almost_equal
+import numpy.lib.recfunctions as rfn
+
+from qinfer.tests.base_test import (
+    DerandomizedTestCase, 
+    ConcreteDifferentiableModelTest,
+    ConcreteModelTest,
+    ConcreteSimulatableTest
+)
+import abc
+from qinfer import (
+    SimplePrecessionModel, SimpleInversionModel,
+    CoinModel, NoisyCoinModel, NDieModel,
+    RandomizedBenchmarkingModel,
+    PoisonedModel, BinomialModel, MultinomialModel,
+    MLEModel, RandomWalkModel,
+    NormalDistribution,
+    BetaDistribution, UniformDistribution,
+    PostselectedDistribution,
+    ConstrainedSumDistribution
+)
+from qinfer.ale import ALEApproximateModel
+from qinfer.tomography import TomographyModel, DiffusiveTomographyModel, pauli_basis, GinibreDistribution
```

Comment:
It should be possible to use [``unittest.skipIf``](https://docs.python.org/2/library/unittest.html#unittest.skipIf) to guard tests which depend on QuTiP, such that they won't be run unless QuTiP is installed and is at least version 3.2.0.